packages/run-protocol/src/vpool-xyk-amm/multipoolMarketMaker.js

```diff
@@ -7,6 +7,7 @@ import { AssetKind, makeIssuerKit } from '@agoric/ertp';
 import { CONTRACT_ELECTORATE, handleParamGovernance } from '@agoric/governance';
 
 import { assertIssuerKeywords } from '@agoric/zoe/src/contractSupport/index.js';
+import { E } from '@agoric/eventual-send';
```

Comment:
The new norm since #3833 is:

```suggestion
import { E } from '@agoric/far';
```